src/components/ResolutionModal.css

```diff
@@ -1,3 +1,7 @@
 .modal-90w {
   width: 30%;
 }
+
+button {
```

Comment:
A note on how css works:
It's not scoped, so this doesn't just affect buttons in the ResolutionModal component. It affects all buttons in the app, and depending where how the app gets used, possibly buttons outside of our app as well.